### PR TITLE
converted the bmp file into c++

### PR DESCRIPTION
**Pull request by HembramBeta777**

### CODE REVIEW

File: bmp.h [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+typedef class Color
+{
+    public:
+
+    BYTE  rgbtBlue;
+    BYTE  rgbtGreen;
+    BYTE  rgbtRed;
+} __attribute__((__packed__))
+RGBTRIPLE;
```
Comment (Himanshu40) [inline bmp.h:55]: Don't use typedef in class. Instead, start like this
```cpp
class className {
public:
// do things
};
```
Read [Struct vs Class](https://www.geeksforgeeks.org/structure-vs-class-in-cpp/), [Typedef](https://en.cppreference.com/w/cpp/language/typedef)

**Review by Himanshu40 — CHANGES_REQUESTED** [review]

File: bmp.h (hunk 1 above)
Comment (HembramBeta777) [reply bmp.h:55]: ok I will update asap🙂

**Comment by Himanshu40** [comment]
I have deleted settings.json, launch.json because these are files auto-generated by VSCode. So when adding files be aware of what you are adding. Never add VSCode settings files to pull request. @HembramBeta777

**Comment by Himanshu40** [comment]
@HembramBeta777 You forgot to add `;` after every class end. You should add it.

**Review by Himanshu40 — APPROVED** [review]

**Comment by Himanshu40** [comment]
I will test it in my PC then if something goes wrong, I will point it out otherwise I will merge it.

**Comment by HembramBeta777** [comment]
> I will test it in my PC then if something goes wrong, I will point it out otherwise I will merge it.

Ok  Sir 😀 @Himanshu40

**Comment by Himanshu40** [comment]
@HembramBeta777 follow this style:-
```cpp
class A {
    public:
        int a;
        long b;
        char c;
} __attribute__((__packed__));
```

File: bmp.h (hunk 1 above)
Comment (chandrakant100) [reply bmp.h]: > ok I will update asap🙂

good

**Review by Himanshu40 — APPROVED** [review]

**Comment by Himanshu40** [comment]
Conversion of struct to class for bmp.h. #2

**Pull request merged**